static/templates/settings/account-settings.handlebars

```diff
@@ -51,33 +51,34 @@
                         {{#if page_params.is_admin}}style="display:none"{{else}}{{#unless page_params.realm_name_changes_disabled}}style="display:none"{{/unless}}{{/if}}
                         title="{{t 'Changing your name has been disabled by your Zulip organization administrators. Contact an administrator for help.' }}"/>
                     </div>
-                    <div id="change_full_name_modal" class="modal modal-bg hide" tabindex="-1" role="dialog"
-                        aria-labelledby="change_full_name_modal_label" aria-hidden="true">
-                        <div class="modal-header">
-                            <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
-                            <h3 id="change_full_name_modal_label">{{t "Change full name" }}</h3>
-                        </div>
-                        <div class="modal-body">
-                            <div class="input-group full_name_change_container">
-                                <label for="email">{{t "New full name" }}</label>
-                                <input type="text" name="full_name" value="{{ page_params.full_name }}" autocomplete="off" spellcheck="false" autofocus="autofocus"/>
-                            </div>
-                        </div>
-                        <div class="alert change_full_name_info"></div>
-                        <div class="modal-footer">
-                            <button class="button white rounded" type="button" data-dismiss="modal">{{t "Cancel" }}</button>
-                            <button id='change_full_name_button' class="button rounded sea-green" data-dismiss="modal">{{t "Change" }}</button>
+                </div>
```

Comment:
I noticed the other forms follow the format of 
```
<form>
    <div1>
    <div2>
```
while this was 
```
<form>
    <div1>
        <div2>
```
so I made it consistent by changing it to the first format.